<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="xLineEditCoordinate"/>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QFrame" name="frameMap">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="yLineEditCoordinate"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="mpActionZoomIn"/>
    <addaction name="mpActionZoomOut"/>
    <addaction name="mpActionPan"/>
    <addaction name="mpLocate"/>
   </widget>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mpActionZoomIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/mActionZoomIn.png</normaloff>:/mActionZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mpActionZoomOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/mActionZoomOut.png</normaloff>:/mActionZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mpActionPan">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/mActionPan.png</normaloff>:/mActionPan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="mpLocate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/mActionAddLayer.png</normaloff>:/mActionAddLayer.png</iconset>
   </property>
   <property name="text">
    <string>Locate</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
